### DISCUSSION TITLE
New install - initial server 500 error

### DISCUSSION THREAD

**Discussion by jvanas** [Kovah/LinkAce#572] [Support] [answered]
Brand new install on CentOS 7, PHP 8.0, and linkace-v1.10.5. (non-docker install)

Never used this before, so sorry if this is an easy one.  I am sure I have just setup something wrong.

When I connect to the web page to "run the setup" I am getting this error:
LinkAce
500 - Server Error
An internal server error occured. If you are the administrator, consult the application logs for details.

I could use some guidance.  No error in my HTTP logs, /var/log/messages and I can't seem to locate logs in LinkAce.

Where do I go from here?

TIA!
-Josh

**Comment by Kovah** [comment, marked as answer]
Logs can be found in the `storage/logs` folder in the LinkAce directory. Is there anything inside?
If not, please replace `APP_DEBUG=false` with `APP_DEBUG=true` and try again. It might show more details about the error.

**Reply by jvanas** [reply]
Thanks! 
That log folder is empty.

What file do I enable debugging in?

**Reply by Kovah** [reply]
Oh, forgot to add. In the .env file.

**Reply by jvanas** [reply]
Thanks!
Still no files in storage/logs after adding the debug.

**Reply by jvanas** [reply]
OK, progress.  Disabled SELinux.  Got things moving.  Now, after setup starts, it dies at the URL: https://server/setup/start

with this displayed in the browser:
Not Found
The requested URL /setup/start was not found on this server.

**Reply by jvanas** [reply]
ok, weird, I had to run the URL:  https://server/index.php and then it redirected to https://server/index.php/setup/start

**Reply by Kovah** [reply]
> The requested URL /setup/start was not found on this server.

This doesn't look right. Whatever web server you are using, there must be a configuration issue. Do you use Apache? If yes, make sure the PHP and mod_rewrite modules are enabled and activated properly.

**Reply by jvanas** [reply]
Yep, needed to set AllowOverride All

Thanks!

**Reply by jvanas** [reply]
Now I can't create the first user.  After it prompts me for username, email, and password, I get:

SQLSTATE[HY000] [1045] Access denied for user 'linkace'@'localhost' (using password: YES) (SQL: select count(*) as aggregate from `users` where `email` =

**Reply by jvanas** [reply]
config.php still had the default password.  fixed.

**Reply by jvanas** [reply]
Thank you so much for all you help!  love this application!

**Reply by Kovah** [reply]
Glad to know that it now works. Let me know when there are any other issues.
